Conference-Android/app/src/main/res/layout/conference_detail.xml

```diff
@@ -4,13 +4,15 @@
     xmlns:tools="http://schemas.android.com/tools"
```

Comment:
On lower-res devices like my phone, the hardcoded pixel height of this activity prevented me from using the activity.  This change adds layout weights to allow for more proportional spacing.  I can show you tomorrow with my phone.